DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/queryOverview/metadata.json

```diff
@@ -0,0 +1,196 @@
+{
+    "Slug": "query-overview",
+    "SourceFileName": "QueryOverview.java",
+    "Title": "Query Overview",
+    "NonInteractive":  true,
+    "DescriptionLines": [
+        "RavenDB has rich querying and indexing support.",
+        "",
+        "**Query the database to find documents that match a particular predicate**.",
+        "The server will return only the matching documents.",
+        "",
+        "The returned documents are loaded and tracked by the Session (except for projections).",
+        "",
+        "From the session, query the database by using any of:",
+        "<p>* session.query (shown in this demo)<br>",
+        "* sesion.advanced.documentQuery<br>",
```

Comment:
I think it is worth to mention that session.advanced.documentQuery is alias to session.query. 
Btw. you have typo here: sesion -> session

Reply:
1. fixed typo
2. done - mentioned aliasing
   -and-
    one of the assets links is to the "Query vs DocumentQuery" article

Reply:
on second thought - having the asset link is enough - since it is too much info on this short into paragraph - doesn't look that good. 
I placed the link higher in the list.